### ISSUE TITLE
Update: 3rd April 2024 → "react-beautiful-dnd: where to from here?"

### ISSUE THREAD

**Issue by alexreardon** [atlassian/react-beautiful-dnd#2573]
Hey all,

I am the author of `react-beautiful-dnd` 👋. It feels really good to engage with this repo again after having poured a few years of my life into it. 

I have an update to share with y'all about `react-beautiful-dnd` and [Pragmatic drag and drop](https://github.com/atlassian/pragmatic-drag-and-drop) (a new drag and drop framework of ours)

<br>
<br>

<div align="center">
<a href="https://www.youtube.com/watch?v=uySl2yiUYE4"><img src="https://github.com/alexreardon/files/assets/2182637/bb7740a3-875c-49aa-be54-e7268d836d60" width="400px"></a>
<br/>
<a href="https://www.youtube.com/watch?v=uySl2yiUYE4"><strong>📺 Watch on Youtube</strong></a>
</div>

**Comment by Galkon** [comment]
Been following the journey on twitter, excited to see this released! Thanks for the amazing contributions!

**Comment by masudhossain** [comment]
React native support?

**Comment by alexreardon** [comment]
> React native support?

No. Pragmatic drag and drop uses web platform drag and drop functionality, and will work for web browsers on iOS and Android. However, it does not call native code directly (eg [UIKit drag and drop](https://developer.apple.com/documentation/uikit/drag_and_drop)) and so is not an appropriate tool for react native.

> is there any benefits of going to Pragmatic with @atlaskit/pragmatic-drag-and-drop-react-beautiful-dnd-migration if react-beautiful-dnd is working out fine for us?

- The migration layer allows you to remove your dependency on `react-beautiful-dnd` (and on it's dependencies, such as `redux` and `react-redux`). 
- If you are migrating to Pragmatic drag and drop in other places, the migration layer can help make `react-beautiful-dnd` experiences feel more like Pragmatic drag and drop experiences - helping overall visual consistency
- There are some nice performance benefits on offer.

All that said, if you are not interested in moving to Pragmatic drag and drop more fully in the longer term, then the migration layer is likely not the right option for you. The migration layer is more of a stepping stone to a full migration.

I hope that helps @masudhossain!

**Comment by ankit-sapkota555** [comment]
Will the migration work for NextJS?

If I want to fully migrate a simple dnd list how much time should it take?

**Comment by ArthurGuez** [comment]
Hi there,
Will there be a migration guide from `react-beautiful-dnd` to `pragmatic-drag-and-drop` ?
Thanks

**Comment by masudhossain** [comment]
I agree, the blue line and no animation feels like a terrible user experience.

**Comment by elron** [comment]
Can't wait for it to work with Svelte. I tried to make it work with Svelte but couldn't do that. Will there be a tutorial/documentation?

**Comment by NikolaMitic-Softelm** [comment]
Any estimation when will we have tutorial for manual migration that you have advised us for?

**Comment by bernaferrari** [comment]
I've been happily using Framer Motion, because this is dead and the replacement is awful.
